### ISSUE TITLE
Ansible Hygene WARN

### ISSUE THREAD

**Issue by usrbinkat** [ContainerCraft/ansible#7]
```
TASK [containercraft.kubespray.packages : Enable Automatic Updates - Auto Apply = True] *****************************************************************************************
[WARNING]: Consider using the replace, lineinfile or template module rather than running 'sed'.  If you need to use command because replace, lineinfile or template is
insufficient you can add 'warn: false' to this command task or set 'command_warnings=False' in ansible.cfg to get rid of this message.
changed: [node1]
```

**Comment by jbpratt** [comment]
should be resolved by #10 successfully publishing on merge

**Comment by jbpratt** [comment]
confirmed
```
TASK [containercraft.kubespray.packages : Enable Automatic Updates - Auto Apply = True] *****************************************************************************************
ok: [node1]
Wednesday 17 November 2021  22:59:07 +0000 (0:00:00.752)       0:00:48.572 ****
```
